WordPress/Classes/ViewRelated/Jetpack/Branding/Coordinator/JetpackBrandingCoordinator.swift

```diff
@@ -6,7 +6,28 @@ class JetpackBrandingCoordinator {
     static func presentOverlay(from viewController: UIViewController, redirectAction: (() -> Void)? = nil) {
 
         let action = redirectAction ?? {
-            // TODO: Add here the default action to redirect to the jp app
+            #if DEBUG
+            let jetpackScheme = "jpdebug"
+            #elseif INTERNAL_BUILD
+            let jetpackScheme = "jpinternal"
+            #elseif ALPHA_BUILD
+            let jetpackScheme = "jpalpha"
+            #else
+            let jetpackScheme = "jetpack"
+            #endif
+
+            guard AppConfiguration.isWordPress,
```

Comment:
Is the overlay going to show in Jetpack? This may be a redundant check but it also isn't hurting anything.

Reply:
Yeah, I guess you're right. This should only ever be called from WP, and the less code the better.